packages/language-server/src/server.ts

```diff
@@ -135,7 +135,11 @@ export function startServer(options?: LSOptions) {
         pluginHost.register(
             new TypeScriptPlugin(
                 configManager,
-                new LSAndTSDocResolver(docManager, workspaceUris, configManager)
+                new LSAndTSDocResolver(
+                    docManager,
+                    workspaceUris.map((uri) => pathToUrl(urlToPath(uri) ?? '')),
```

Comment:
There's `normalizeUri`, any reason why you didn't use that one?

Reply:
Totally didn't remember there's this function lol. No wonder it only happens here.